src/main/java/ch/njol/skript/effects/EffDetonate.java

```diff
@@ -0,0 +1,49 @@
+package ch.njol.skript.effects;
+
+import org.bukkit.entity.Firework;
+import org.bukkit.entity.Projectile;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+
+@Name("Detonate Entity")
+@Description("Automtically Detonates an entity")
```

Comment:
```suggestion
@Description("Automtically detonates an entity")
```
You should also mention what entities it detonates, right now it currently only detonates a firework (not sure if you'll add more), which is not mentioned anywhere but the example

Reply:
I added support for WindCharge and Minecarts as well